src/index.html

```diff
@@ -1,13 +1,56 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="container">
+      <div class="container__card card" data-qa="card">
+        <img src="images/imac.jpeg" class="card__img" alt="asd"></img>
+        <p class="card__name">
+            Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </p>
+        <p class="card__code">
+            Код товара: 195434
+        </p>
+        <div class="card__rating rating">
+          <div class="rating__stars stars">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="rating__comment">
+              Отзывов: 5
+          </p>
+        </div>
+        <div class="card__price price">
+          <p class="price__text">
+              Цена:
+          </p>
+          <p class="price__num">
+              69 999 грн
+          </p>
+        </div>
```

Comment:
There is no need of additional class price, you can style `card__price` and write interanl classes as `card__price-text` etc.
```suggestion
        <div class="card__price">
          <p class="card__price-text">
              Цена:
          </p>
          <p class="card__price-num">
              69 999 грн
          </p>
        </div>
```